### ISSUE TITLE
Requirements document?

### ISSUE THREAD

**Issue by kixelated** [kixelated/warp-draft#45]
Do you guys think we should have an explicit document, or use Github issues to track requirements?

**Comment by acbegen** [comment]
I think a separate requirements draft is needed. The current requirements document is more like use-cases document. I also think there is a need for a "definitions" document to get everyone on the same page and keep the terminology consistent. I am volunteering for this work if anybody else is willing to help.

**Comment by VMatrix1900** [comment]
@acbegen I agree a definition and a requirement draft are needed and I am willing to help.

**Comment by kixelated** [comment]
I'm going to work on a definitions document next, since naming has been the most problematic. I'll need a lot of help with non HLS/H.264 terminology.

**Comment by suhasHere** [comment]
@kixelated I am happy to help on this

**Comment by SpencerDawkins** [comment]
I have two thoughts here - 
- the use cases and requirements document that @fiestajetsam and I have been working on is intended to contain "requirements", but I suspect that document should be talking about requirements that enable MOQ to be used in overall systems (so, things like the ability to have "relays" that don't have access to the media payload, as well as the ability to have relays that are also, for example, transcoders that need access to the media payload). 

I suspect that protocol drafts, such as this one, also need to talk about more detailed requirements that apply to a single protocol. Does that make sense?
- I could not agree more with @acbegen, @VMatrix1900, @kixelated, and @suhasHere about the need for a definitions draft, and I'm **also** willing to help. Has anyone started that draft yet? By "started", I mean, is there an active GitHub repo for it?

**Comment by afrind** [comment]
I think the ship has sailed on requirements documents.  We have the requirements draft, and we have architectural issues in this repo -- ready to close?
